### PR TITLE
xGov-168: Pact.fi Marketing Initiatives

### PR DESCRIPTION
**Pull request by tulustul**

### CODE REVIEW

File: Proposals/xgov-168.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+---
+id:
```
Comment (SudoWeezy) [inline Proposals/xgov-168.md:2]: id should be 168

File: Proposals/xgov-168.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+period: 168
+title: Pact.fi Marketing Initiatives
+author: Michael Cotton <mc@optio.capital>, AJ Milne
+discussions-to: https://github.com/algorandfoundation/xGov/pull/168
+company_name: Pact
+category: dApps
+focus_area: DEX
```
Comment (SudoWeezy) [inline Proposals/xgov-168.md:9]: DEX should be Dex

**Review by SudoWeezy — COMMENTED** [review]
Please update before the end of the month :)

**Comment by nonfungibleab** [comment]
Proposer, please post your algorand forum discussion thread link here. Thank you.

**Comment by nonfungibleab** [comment]
Hi @tulustul congrats on your proposal approval! We need your email address to start the contract process.

**Pull request merged**